Java compile fails in GitHub codespaces

Building in GitHub codespaces (via the "compile" lifecycle component provided with the maven extension) fails with the following:

>[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project jt400: Execution default-compile of goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile failed: syntax error @[28,16] in file:/workspaces/JTOpen/src/main/java/jt400Native/module-info.java -> [Help 1]
[ERROR] 

The root cause, I believe, is that the `module-info.java` files are still gated with the `JAVA9` ifdef. GitHub codespaces is building with Java 11 by default. 
So the "base" compile tries to build a part that is just one big comment. 

Since `module-info.java` is only built in the appropriate distribution builds (pom-dist.xml), I think the solution here is to simply remove the JAVA9 gate from these parts. But more investigation is needed. I'm not sure why the CI is passing.

--- a/pom.xml
+++ b/pom.xml
@@ -144,6 +144,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.10.0</version>
+        <executions>
+          <execution>
+            <id>default-compile</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+            <configuration>
+              <source>7</source>
+              <target>7</target>
+              <fork>true</fork>
+              <excludes>
+                <exclude>**/module-info.java</exclude>
+              </excludes>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
     </plugins>
   </build>
 </project>

